src/components/Widgets/StatPing.vue

```diff
@@ -73,6 +77,15 @@ export default {
     endpoint() {
       return `${this.hostname}/api/services`;
     },
+    groupId() {
+      return this.options.groupId || 0;
+    },
+    showChart() {
+      return typeof this.options.showChart !== 'boolean' ? true : this.options.showChart;
```

Comment:
It's sometimes easier to have options that default to false, e.g. `hideChart` instead of `showCart`, as then we could just evaluate it with just

```javascript
hideChart() {
  return !!this.options.hideChart;
},
```

But this is fine too, your way is probably more logical for the user also :)